packages/xpetra/src/Map/Xpetra_Map.hpp

```diff
@@ -75,14 +75,29 @@ namespace Xpetra {
 # endif
 #endif
 
+  namespace Details
+  {
+    //Check for the default Tpetra GlobalOrdinal (long long)
+    #ifdef HAVE_XPETRA_TPETRA
```

Comment:
See above.  Also, perhaps we should consolidate these two "test the macros" things in a single place.